sandbox/OPEN-SOURCE.md

```diff
@@ -0,0 +1,29 @@
+# Open Source
+
+Defense Unicorns values Open Source. We understand that Open Source is a way for all members of the company to prevent silos and work in a collaboration-first mentality. 
+
+## Advocates
+- Brandt Keller
+
+## Why?
+
+Open Source directly contributes to the company vision:
+
+> A free and secure world that enables progress and innovation
+
+Progress and Innovation are spaces that any member of Defense Unicorns can strive towards.
+It is also a space where we can contribute to the betterment of society through knowledge sharing and teaching others. These others may often be other Uncicorns who are baordening their horizions to new technologies. It may also be external, and the sharinf of processes, information, data and code for the purpose of building a more secure tomorrow. 
+
+## Internal Impact
+Advocating Open Source as a company value means promoting collaboration to all facets of company operation in open and traceable manners. Asynchronous collaboration and documentation allow traceability to be a forcing function for the members of the company to understand current thoughts and provide historical reference for when something needed to change. This allows all members of the company to understand and provide valuable context and perspectives that may not have been originally established.
+
+We share processes in such a way that enable members to provide contributions directly and asynchronously - rather than through a specified focal and/or non-documented synchronous engagements. 
+
+It also means that we advocate and encourage any member of the company to review and participate in current established products/processes and enable them to contribute through any iteration that makes the product/process better than it was originally. 
+
+## External Impact
+
+Open Source advocacy external to the company means engaging and enabling for knowledge to be spread in public forums only for the purpose of educating others. This aligns with the flywheel and establishing credibility for the processes and products that drive the flywheel. This can be engaging in communities of practice in order to provide domain knowledge to others. This can also be through events such as conferences and driving conversation around experiences. These prove to external entities that we are a collaborative company and truly believe in the company vision. 
```

Comment:
I would argue that this does go beyond mere education.  It allows them to see and then take/remix ideas for their own use.  One example that I was fond of previously was "The Apache Way" which I borrowed ideas from (such as their form of lazy consensus voting) to improve government change management processes and default to yes more often.

Reply:
IMO others should be free to rip repos like this cultural document and form their own companies around it and continue experimenting.  We as a company then benefit from that because we can look at their experiments and see if we can implement some of it too.

Another example of that is how "ATO in 30 days" became "ATO in a day" became "Continuous Security" became the "Continuous ATO" between 18F, NGA and the US Air Force.  Those ideas were freely shared and built upon to create better models for doing business.